### PR TITLE
native/ui/efl: Delete description of unsupported features

### PR DESCRIPTION
**Pull request by JSUYA**
### Change Description ###
In edc, SOURCE keyword is not yet supported for max,min of image part.
Therefore, delete description related to SOURCE.

### CODE REVIEW

**Comment by JSUYA** [comment]
@mijin85cho Hi Please check this PR.

**Comment by mijin85cho** [comment]
@annie-samsung Would you please review this PR?
We are waiting your comment.

File: docs/application/native/guides/ui/efl/learn-edc-part.md [hunk 1]
```diff
@@ -701,17 +701,17 @@ part {
 
     Sets the minimum size calculation. See the `edje_object_size_min_calc()` and `edje_object_size_min_restricted_calc()` functions of the Edje Object Geometry API (in [mobile](../../../api/mobile/latest/group__Edje__Object__Geometry__Group.html) and [wearable](../../../api/wearable/latest/group__Edje__Object__Geometry__Group.html) applications). The value tells the min size calculation routine where this part does not change size in width or height (1 for it does not, 0 for it does), so the routine does not try to expand or contract the part.
```
Comment (annie-samsung) [inline docs/application/native/guides/ui/efl/learn-edc-part.md:702]: Suggest a rephrase, please check if the below sentence is technically accurate?

The value tells the minimum size calculation routine where this part does not change size in width or height, so the routine does not try to expand or contract the part. The value 1 indicates it does not change size, and the value 0 denotes there is size change.


min in code tag or change to minimum

File: docs/application/native/guides/ui/efl/learn-edc-part.md [hunk 2]
```diff
@@ -701,17 +701,17 @@ part {
...
 
-  - `min [width] [height] or SOURCE`
+  - `min [width] [height]`
 
-    Sets the minimum size of the state. When the property is set to `SOURCE`, it looks at the original image size and enforces its minimal size to match at least the original one. The part must be an `image` or a `group` part.
+    Sets the minimum size of the state.
 
   - `minmul [width multiplier] [height multiplier]`
 
     Sets the multiplier forcibly applied to whatever minimum size is only during minimum size calculation.
```
Comment (annie-samsung) [inline docs/application/native/guides/ui/efl/learn-edc-part.md:710]: This is not clear can this be rephrased?

**Comment by JSUYA** [comment]
HI @annie-samsung 
Thanks for your comment. But you reviewed the code I haven't edited.
The content I have modified is about the SOURCE of `min` and `max`.
Are there any problems with my changes? Please check it again.

File: docs/application/native/guides/ui/efl/learn-edc-part.md [hunk 3]
```diff
@@ -701,17 +701,17 @@ part {
...
 
-  - `max [width] [height] or SOURCE`
+  - `max [width] [height]`
 
-    Sets the maximum size of the state. A size of -1.0 means that it is ignored in one direction. When the property is set to `SOURCE`, Edje enforces the part to be not more than the original image size. The part must be an `image` part.
+    Sets the maximum size of the state.
```
Comment (annie-samsung) [inline docs/application/native/guides/ui/efl/learn-edc-part.md:714]: A size of -1.0 means that it is ignored in one direction. Is this information associated with SOURCE or valid even now?

**Comment by annie-samsung** [comment]
> HI @annie-samsung
> Thanks for your comment. But you reviewed the code I haven't edited.
> The content I have modified is about the SOURCE of `min` and `max`.
> Are there any problems with my changes? Please check it again.

@JSUYA, okay, I have a query regarding the deletion, please find my comment for the same.

File: docs/application/native/guides/ui/efl/learn-edc-part.md (hunk 3 above)
Comment (JSUYA) [reply docs/application/native/guides/ui/efl/learn-edc-part.md]: Thanks for the comment. It's my fault. I added a new patch to recover this.

**Review by annie-samsung — APPROVED** [review]
Thanks for fixing the comments, approving.

**Pull request merged**